### DISCUSSION TITLE
Hard to find this library

### DISCUSSION THREAD

**Discussion by batata004** [fomantic/Fomantic-UI#365] [General] [answered]
Hi,

If you go to google (for example) and search only for "fomantic ui" it's impossible to find the FOMANTIC UI GITHUB page not even at the third page of results! I dont know if you need some SEO expert to help into this but it's freaking hard to find FOMANTIC UI in the internet. I just wanted to check the latest release notes and had to track the fomantic repository link in a forum discussion outside github.

**Comment by idmarinas** [comment]
This has an easy solution, use Bing. :laughing:

Google interprets  "Fomantic UI " As you have been mistaken when writing and that your intention was to use "Semantic UI ". But to ignore that, you can give it to  "Search instead for fomantic UI"

With Bing, not have this problem.

Maybe this is not solved only with SEO.

**Comment by lubber-de** [comment, marked as answer]
Google will solve this on his own, if you help us spread the word for FUI :wink: 
If many, many backlinks to the FUI repo and/or website will occur on the net, the Google algorythm will recognize that. Also when the FUI Repo gains lot more Stars

- If you are a SEO specialist: We accept PRs for https://github.com/fomantic/Fomantic-UI-Docs which represents https://fomantic-ui.com
- If you are using FUI on your website: You may want to consider linking to the repo like "Made with :heart: and Fomantic UI"
- If you are a developer-teammate or leader: Tell your colleagues!
- If you still (have to) use SUI: Branch your repo using FUI, try it out, deploy on a testserver to convince your colleagues or show them, they could probably remove lots of workarounds, because FUI finally has fixed so many bugs which are jailed unfixed in SUI for many years
- If you are blogging: Tell the world about FUI and its benefits and how or why you should use it and link it!
- If you are youtubing: Record a cool series of "how to" live coding sessions!
- If you are twittering: Spread your knowledge, post our releases, share your love with FUI
- If you are instagramming: Do the same having your girl-/boyfriend posing infront of the camera next to a monitor showing the latest FUI features!
- And if you are facebooking: Do all of the above (but make sure you don't get thousands of date-invites, because people might think you mean "romantic" instead)
:slightly_smiling_face:
